src/InputGuess.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import Input from "@mui/material/Input";
+import { Button } from "@mui/material";
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userWord: "",
+    };
+  }
+
+  handleChange = (e) => {
+    let { value } = e.target;
+    if ((value.match(/^[a-z]+$/) && value.length === 1) || value === "") {
```

Comment:
```suggestion
    if ((value.match(/^[a-z]+$/) && value.length === 1) || !value) {
```

empty strings are falsy in JavaScript